cellarium/cas/client.py

```diff
@@ -538,6 +557,136 @@ def annotate_10x_h5_file(
             include_dev_metadata=include_dev_metadata,
         )
 
+    def annotate_matrix_with_cell_type_statistics_strategy(
+        self,
+        matrix: t.Union[str, "anndata.AnnData"],
+        chunk_size=CHUNK_SIZE_ANNOTATE_DEFAULT,
+        cas_model_name: str = "default",
+        count_matrix_input: constants.CountMatrixInput = constants.CountMatrixInput.X,
+        feature_ids_column_name: str = "index",
+        feature_names_column_name: t.Optional[str] = None,
+        include_extended_statistics: bool = True,
+    ):
```

Comment:
Should define a return type here.  I think this?
```suggestion
    ) -> t.List[t.Dict[str, t.Any]]:
```